test/variational/test_exact_state.py

```diff
@@ -54,6 +56,12 @@
     kernel_init=normal(stddev=0.1),
     hidden_bias_init=normal(stddev=0.1),
 )
+machines["model:(C->C)"] = RBM(
+    alpha=10,
+    dtype=complex,
+    kernel_init=normal(stddev=1.0),
+    hidden_bias_init=normal(stddev=1.0),
+)
```

Comment:
Why are you adding another RBM with much higher stddev and alpha? 
Isn't the test by the one above enough?

Reply:
Besides, I think this stddev is too large for `alpha=10`. This might be why tests are failing

Reply:
I run the tests 20 times on my machine and it failed 3 times with `normal(stddev=1.0)` but never with `normal(stddev=0.1)`. I really think this should be decreased.

Reply:
The initial reason why I added another (more complex) machine was that sometimes the derivatives can become relatively small for an initial state which only fluctuates a tiny amount around a uniform state. This way, the gradient checks might go through by accident even if the derivatives are wrong. Just wanted to add a model which is significantly further away from a uniform state just to be sure that the derivatives are right. I do appreciate that the standard deviation was a little bit too large for things still to be numerically stable, will decrease that a bit which should fix things.
(Tests always passed on my machine so I didn't realize these issues before looking at the github test outputs...)